lib/src/models/w3c_header.dart

```diff
@@ -0,0 +1,34 @@
+class W3CHeader {
```

Comment:
Let's use the full name W3C* instead of W3* for all the relevant components in the file.